src/app/controllers/email-submissions.server.controller.js

```diff
@@ -552,19 +552,25 @@ exports.saveMetadataToDb = function (req, res, next) {
 
   // Create submission hash
   let concatenatedResponse = concatResponse(formData, attachments)
-  let submissionLogstring
+  let submissionLog
 
   createHash(concatenatedResponse)
     .then((result) => {
       submission.responseHash = result.hash
       submission.responseSalt = result.salt
-      submissionLogstring = `Saving submission ${submission.id} to MongoDB with hash ${submission.responseHash}`
+      submissionLog = {
+        message: 'Saving submission to MongoDB',
+        submissionId: submission.id,
+        formId: form._id,
+        ip: getRequestIp(req),
+        responseHash: submission.responseHash,
+      }
       // Save submission to database
-      logger.profile(submissionLogstring)
+      logger.profile(submissionLog)
```

Comment:
why are we logging the same submission string twice? Seems like the difference is whether the submission document is `saving` or `saved`. Can we make that clearer?

Reply:
We don't log it twice, `logger.profile` is used to measure the duration between logs, so the log is only made at the second call to `profile` and will contain the time taken between the first and second calls. But as Yuanruo said above, we don't need the profiling code anymore, so I've changed it to one call to `logger.info`.